WinCCOA_QualityChecks/scripts/libs/classes/QualityGates/QgCtrlppCheck/QgCtrlppCheck.ctl

```diff
@@ -162,6 +151,19 @@ class QgCtrlppCheck : QgBase
   /// Checks if the error shall be filtered.
   protected bool isErrorFiltered(const CppCheckError &error)
   {
+    /// @todo it shallbe somehow configurable and
+    /// done in the ctrlppcheck (in cpp source) to eliminate CPU usage
+    if (error.path != error.path0)
+    {
+      // inform only about failures in checked sources.
+      // No body interested about sub-project failures.
+      return TRUE;
+    }
+
+    QgFile f = QgFile(error.path);
+    if ( f.isExample() || f.isTest() || !f.isPatternMatch(includeFilesPattern) )
```

Comment:
```suggestion
    if (f.isExample() || f.isTest() || !f.isPatternMatch(includeFilesPattern))
```
Most adapted code uses C++ coding-guideline, why change here to CTRL guideline?

Reply:
because it is ctrl code? I will provide later (when this changes are merged) on PR, where I will format all the ctrl code with astyle to have consitent formatting-